Palindromos/src/ar/edu/uno/poo2/modulo/Palindromo.java

```diff
@@ -2,11 +2,50 @@
 import java.io.*;
 
 public class Palindromo {
-	
+	private String palabra = null;
+	private String[] palindromos = null;
+
+	public Palindromo(String palabra){
+		this.setPalabra(palabra);
+	}
+
 	public void cortePalabra(String string){
-		
 	}
 
-	
+	public boolean esPalindromo(){
+		String palabra = this.getPalabra();
+		boolean es = false;
+		Integer largo = palabra.length();
+		char[] palabraArray = palabra.toCharArray();
```

Comment:
No conviene trabajar todo como String e ir comparando las letras con el charAt()?
this.palabra.charAt(i)